metatron-app/metatron-discovery
# #3290 separate oauth timeout by clientId

### Description

Separate oauth timeout setting by clientId

**Related Issue** : #3290 



### How Has This Been Tested?

1. Set the values ​​of the ACCESS_TOKEN_VALIDITY and REFRESH_TOKEN_VALIDITY columns of the OAUTH_CLIENT DETAILS table (in second units) for each clientId.
* **Set the values ​​of the ACCESS_TOKEN_VALIDITY and REFRESH_TOKEN_VALIDITY columns the same**.
2. Login metatron Discovery.
3. Check if the timeout time set in step 1 is applied.
4. Log in to the external client using metatron Discovery.
5. Check if the timeout time set in step 1 is applied.

#### Need additional checks?


### Types of changes



### Checklist:


### Additional Context
